pkg/local_object_storage/engine/inhume.go

```diff
@@ -264,14 +264,35 @@ func (e *StorageEngine) processExpiredObjects(addrs []oid.Address) {
 }
 
 func (e *StorageEngine) processExpiredLocks(lockers []oid.Address) {
+	var unlocked, expired []oid.Address
 	for _, sh := range e.unsortedShards() {
-		sh.HandleExpiredLocks(lockers)
+		unlocked = sh.HandleExpiredLocks(lockers)
+		for _, sh2 := range e.unsortedShards() {
+			expired = append(expired, sh2.FilterExpired(unlocked)...)
+		}
+	}
+	ss := []string{}
+	for _, addr := range expired {
+		ss = append(ss, addr.String())
+	}
+	e.log.Debug("expired objects", zap.Strings("addrs", ss))
```

Comment:
can we extend the message a little? i would like to know that we are handling "new" expired objects after some locks were dropped